src/hooks/room/useRoomAvatar.ts

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { useEffect, useState } from "react";
+import { EventType, type MatrixEvent, type Room, RoomStateEvent } from "matrix-js-sdk/src/matrix";
+import { type Optional } from "matrix-events-sdk";
```

Comment:
Lets please not use a WIP-yet-unmaintained project any more than we already do today.